packages/env/lib/commands/destroy.js

```diff
@@ -57,10 +57,14 @@ module.exports = async function destroy( { spinner, debug } ) {
 		return;
 	}
 
+	if ( scripts ) {
+		await executeLifecycleScript( 'beforeDestroy', config, spinner );
```

Comment:
Any thoughts on before vs after? I like the idea of after because what gets set up could be destroyed anyways, which could make you notice that you don't need an extra destroy script